src/test/java/com/hivemq/codec/decoder/MQTTMessageDecoderTest.java

```diff
@@ -174,4 +176,147 @@ public void test_second_connect_received() {
         assertFalse(channel.isOpen());
 
     }
+
+    @Test
+    public void test_connect_mqtt5_packet_size_too_large() {
+        final byte[] mqtt5Connect = {
+                // fixed header
+                //   type, reserved
+                0b0001_0000,
+                // remaining length
+                17,
+                // variable header
+                //   protocol name
+                0, 4, 'M', 'Q', 'T', 'T',
+                //   protocol version
+                5,
+                //   connect flags
+                (byte) 0b0000_0000,
+                //   keep alive
+                0, 0,
+                //   properties
+                0,
+                // payload
+                //   client identifier
+                0, 4, 't', 'e', 's', 't'};
+
+        test_connect_packet_size_too_large(mqtt5Connect);
+    }
+
+    @Test
+    public void test_connect_mqtt3_1_1_packet_size_too_large() {
```

Comment:
please name new tests according to the conventions

Reply:
its there, for some reason github its not updateing the repo files